experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -273,16 +292,28 @@ end
 #
 ###############################################################################
 
+function Base.:(==)(V1::LieAlgebraModule{C}, V2::LieAlgebraModule{C}) where {C<:RingElement}
+  return V1.L == V2.L &&
+         V1.dim == V2.dim &&
```

Comment:
The comparison should be ordered by cost, first the cheap comparisions (dim, s), then the more expensive ones